### ISSUE TITLE
Load average is not propely determined for FreeBSD OS

### ISSUE THREAD

**Issue by denis-konovalyenko** [zold-io/zold#606]
The load average equals to `0`. For more details please refer to the http://91.211.13.180:4096/ response:
```
load: 0
```
And below is an example of actual values:
![screenshot_2018-11-30_16-29-49](https://user-images.githubusercontent.com/15904097/49327175-608b2080-f566-11e8-98da-8da7ccf8ab40.png)

It is important to have the load value right as the node stability might be under the risk of not even being just overwhelmed by processing new requests, which is not far from the truth according to the queue length values, but rather causing the whole network to suffer from such speed degradation. The related issue is #544.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@denis-konovalyenko[/z](https://www.0crat.com/u/denis-konovalyenko) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/CAZPZR9FS) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by yegor256** [comment]
@denis-konovalyenko you are right, we don't get the "load average" value right in FreeBSD. This is the limitatio of this library we are using: https://github.com/rderoldan1/usagewatch_ext

**Comment by 0crat** [comment]
Job [#606](https://github.com/zold-io/zold/issues/606) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @denis-konovalyenko[/z](https://www.0crat.com/u/denis-konovalyenko)
